sphinx_js/templates/function.rst

```diff
@@ -10,4 +10,11 @@
 
    {{ content|indent(3) }}
 
+   {% if examples -%}
+   :Examples:
+   {% for example in examples -%}
+     .. code-block:: js
 
+     {{ example|indent(5) }}
```

Comment:
Is 5 special? I see we used 3 in other places where we had to indent.

Reply:
It's because I further indented the body of the `for` directive, so it needs an extra 2 spaces of indentation for everything to line up, with the `.. code-block::`. Do you think it's better to dedent it to stay consistent with the rest of the template?